<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramAnalyzerWidget</class>
 <widget class="QWidget" name="HistogramAnalyzerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogram</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/icons/view-object-histogram-linear.png</normaloff>:/icons/view-object-histogram-linear.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="SourcesSelectionWidget" name="sourcesSelection"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="logarithmicCheckBox">
          <property name="text">
           <string>Logarithmic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="HistogramViewer" name="histogramViewer" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="luminancePushButton">
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/histcolor-luminance.png</normaloff>:/icons/histcolor-luminance.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redPushButton">
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/histcolor-red.png</normaloff>:/icons/histcolor-red.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="greenPushButton">
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/histcolor-green.png</normaloff>:/icons/histcolor-green.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bluePushButton">
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/icons/histcolor-blue.png</normaloff>:/icons/histcolor-blue.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourcesSelectionWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">SourcesSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistogramViewer</class>
   <extends>QWidget</extends>
   <header location="global">HistogramViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
